.travis/after_success_static.sh

```diff
@@ -5,38 +5,16 @@ cd "$TRAVIS_BUILD_DIR"
 
 rm -rf .static
 mkdir .static
+mkdir .static/cmf
 mkdir .static/components
 mkdir .static/containers
 mkdir .static/forms
 mkdir .static/icons
+mkdir .static/logging
```

Comment:
No it's because the coverage folder goes in `.static/<package-name>`
So the coverage of logging will be moved to `.static/logging`, that's why we create this folder